STTBot/events/app_mention.py

```diff
@@ -27,6 +27,12 @@ def _handle_user_mention(client, event_data, say):
 
     matching_cmd = [command for command in commands if command['cmd'] == cmd and command['sub_cmd'] == sub_cmd]
 
+    # Check whether sub_cmd could be an argument.
+    if len(matching_cmd) == 0:
+        matching_cmd = [command for command in commands if command['cmd'] == cmd and len(command['args']) > 0]
```

Comment:
This is fine for now but will break if we add another pin command which takes an argument instead of a sub_cmd.

e.g. if we add a command `@STTBot pin <user>` to get pins from a certain user, matching_cmd will be of length 2 and line 40 will just pick the first match instead of the correct one.